src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/AppointmentBookingAPITest.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.restassured.RestAssured;
+import org.apache.http.entity.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.justice.digital.delius.JwtAuthenticationHelper;
+import uk.gov.justice.digital.delius.JwtParameters;
+import uk.gov.justice.digital.delius.controller.wiremock.DeliusApiExtension;
+import uk.gov.justice.digital.delius.controller.wiremock.DeliusApiMockServer;
+import uk.gov.justice.digital.delius.data.api.AppointmentCreateRequest;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(SpringExtension.class)
```

Comment:
You could alternatively also extend with `DeliusApiExtension` rather than creating mock server manually